Extensions/InvariantCultureTypeConversionExtensions.cs

```diff
@@ -12,6 +12,15 @@ public static class InvariantCultureTypeConversionExtensions
         /// <returns>A predictable string representation of the given <paramref name="value"/>.</returns>
         public static string ToInvariantString(this int value) => value.ToString(CultureInfo.InvariantCulture);
 
+        /// <summary>
+        /// Returns the string representation of the given <paramref name="value"/> using the <see
+        /// cref="CultureInfo.InvariantCulture"/>.
+        /// </summary>
+        /// <param name="value">The value to convert to a string.</param>
+        /// <returns>A predictable string representation of the given <paramref name="value"/>.</returns>
+        public static string ToInvariantString(this int? value) =>

```

Comment:
Shouldn't these extensions be in HL instead of HE? For example their OC counterparts are [here](https://github.com/Lombiq/Helpful-Libraries/blob/dev/Lombiq.HelpfulLibraries.Common/Extensions/NumberExtensions.cs). If there is some historic reason for them to be in HE, might be worth adding a comment above the class.